ui/src/app/components/workflows/workflow/dynamic-parts/key-string-value-part/key-string-value-part.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I think this component needs some non-trivial testing, but can be done in a different PR

Reply:
Yes, https://github.com/AbsaOSS/hyperdrive-trigger/issues/143